cfme/automate/simulation.py

```diff
@@ -1,14 +1,12 @@
 # -*- coding: utf-8 -*-
-from cfme.utils.appliance import get_or_create_current_appliance
 from cfme.utils.appliance.implementations.ui import navigate_to
 
 
 def simulate(
         instance=None, message=None, request=None, target_type=None, target_object=None,
         execute_methods=None, attributes_values=None, pre_clear=True, appliance=None):
     """Runs the simulation of specified Automate object."""
```

Comment:
**OPTIONAL:** Could you extend the doc string to include a description of the parameters and in the case of appliance point out that it is required.